aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -1021,8 +1021,7 @@ def nan_to_num(self):
             raise NotImplementedError("This operation does not make sense for OMX matrices")
 
         if np.issubdtype(self.dtype, np.floating) and self.matrix_view is not None:
```

Comment:
Sometimes it helps for readability to put parenthesis in conditions : 

```python
if np.issubdtype(self.dtype, np.floating) and (self.matrix_view is not None):
```

Reply:
I believe that tools like black will remove unnecessary parentheses... I don't particularly find either better or worse.